### ISSUE TITLE
Remove option assume_homogeneous

### ISSUE THREAD

**Issue by schillic** [JuliaReach/Reachability.jl#466]
I would say that one just pass a homogeneous system. Having such options can just lead to confusion.

**Comment by mforets** [comment]
Yes, using dispatch on homogeneous system (explot `islinear` trait) is the way to go 👍

**Comment by mforets** [comment]
Proposal: rename `asssume_homogeneous` -> `ignore_inputs`.

**Comment by mforets** [comment]
> Yes, dispatch on homogeneous systems (make use of `islinear` trait) is the way to go 👍

This is not "needed", since the dimension of the input set is checked [here](https://github.com/JuliaReach/Reachability.jl/blob/master/src/ReachSets/ContinuousPost/BFFPSV18/reach.jl#L110).
